posthog/api/team.py

```diff
@@ -40,7 +40,7 @@ def has_permission(self, request: request.Request, view) -> bool:
                 # if we're not requesting to make a demo project
                 # and if the org already has more than 1 non-demo project (need to be able to make the initial project)
                 # and the org isn't allowed to make multiple projects
-                "is_demo" not in request.data
+                ("is_demo" not in request.data or not request.data["is_demo"])
```

Comment:
on second thoughts, this is convoluted enough that it's worth splitting these conditions out now, will do so.

Reply:
doesn't this mean that any user can bypass this by adding "is_demo" = False to the request body? Doesn't seem to make sense to me